src/UKHO.ERPFacade.Common/HttpClients/SapClient.cs

```diff
@@ -16,15 +16,25 @@ public SapClient(HttpClient httpClient, IOptions<SapConfiguration> sapConfig)
         {
             _httpClient = httpClient;
             _sapConfig = sapConfig ?? throw new ArgumentNullException(nameof(sapConfig));
+        }
+
+        public async Task<HttpResponseMessage> PostEventData(XmlDocument sapMessageXml, string sapServiceOperation, string userName, string password)
```

Comment:
Replace userName with Username in all places

Reply:
Replaced userName with username.